woocommerce/woocommerce-gateway-stripe
# Use better error messages when an ajax request fails

closes #1991

If an ajax requests fails for many reasons only a message with the text "error" is shown at the top of the page.
This will make 3 better messages.

To test:
Load page with UPE.
Disable network by opening Network tab in chrome debug tools. Change throttling to offline. 
See that the better error message is shown.
![image](https://user-images.githubusercontent.com/1534605/135474120-fd364c65-d5c4-4af4-b1ca-98b2c3bab44e.png)

Feel free to add improved error messages. 
Once this is approved we can backport it to WCPay.
